packages/vkui/src/components/Typography/Typography.module.css

```
@@ -1,9 +1,16 @@
 .Typography--normalize {
-  display: block;
   margin: 0;
   padding: 0;
 }
 
+.Typography--normalize:not(.Typography--inline) {
+  display: block;
+}
+
+.Typography--inline {
+  display: inline-block;
+}
```

```suggestion
.Typography--inline {
  display: inline-block;
}
```

можно опустить `.Typography--normalize:not(.Typography--inline)`, чтобы не усложнять, + чтобы не связывать `noramalize` и `inline`

---

Разобрался почему падают скриншотные тесты: Некоторые компоненты навешивают свои стили на Typography, где переопределяют display. А из-за селектора `.Typography--normalize:not(.Typography--inline)` перебивался стиль display, так как этот селектор имеет больше веса. Выход - сделать селектор с таким же весом. Переделал реализацию

Насчет связывания noramalize и inline. Придется все равно делать какую-то проверку, так как эти пропсы могут быть одновременно